deployment/playbooks/base-install.yml

```diff
@@ -0,0 +1,211 @@
+      command: rm -rf {{ volttron_root }}
+      when: redo_volttron
+
+    - name: make sure hostname folder exists
+      file:
+        path: "{{ ansible_env.HOME }}/{{ inventory_hostname }}"
+        state: directory
+
+    - name: Extract archive
+      unarchive:
+        src: "{{ ansible_env.HOME }}/volttron-{{ inventory_hostname }}.tar.gz"
+        dest: "{{ ansible_env.HOME }}/{{ inventory_hostname }}/"
+        remote_src: yes
+      when: redo_volttron
+
+    - name: rename folder
+      command: mv "{{ ansible_env.HOME }}/{{ inventory_hostname }}/volttron-{{git_branch}}" "{{ volttron_root }}"
+      when: redo_volttron
+
+    - name: Bootstrap the python3 environment
+      command:
+        cmd: "python3 bootstrap.py"
+        chdir: "{{ volttron_root }}"
+        creates: "{{ volttron_root }}/env"
+
+    - name: Test if pika installed
+      shell: "cd {{ volttron_root }} && env/bin/pip list --disable-pip-version-check --format legacy| grep -F gevent-pika | wc -l"
+      args:
+        chdir: "{{ volttron_root }}"
+      register: already_installed_pika
+
+    - name: Boostrap rmq environment
+      when: (messagebus == "rmq" and "0" not in already_installed_pika.stdout)
+      command:
+        cmd: "env/bin/python bootstrap.py --rabbitmq"
+        chdir: "{{ volttron_root }}"
+
+    - name: Test if ws4py is installed (test if web has been installed)
+      raw: "cd {{ volttron_root }} && env/bin/pip list --disable-pip-version-check --format legacy| grep -F ws4py | wc -l"
+      args:
+        chdir: "{{ volttron_root }}"
+      register: web_enabled
+
+    - name: Enabling web packages for environment
+      when: (enable_web is defined and enable_web == True) # and "0" not in web_enabled.stdout)
+      command:
+        cmd: "{{ volttron_root }}/env/bin/python bootstrap.py --web"
+        chdir: "{{ volttron_root }}"
+
+    - name: Enabling master driver packages for environment
+      when: (enable_drivers is defined and enable_drivers == True)
+      command:
+        cmd: "{{ volttron_root }}/env/bin/python bootstrap.py --drivers"
+        chdir: "{{ volttron_root }}"
+
+    - name: Install extra requirements for this instance
+      pip:
+        virtualenv: "{{ansible_env.HOME}}/volttron/env"
+        name: "{{ item }}"
+      with_items: "{{ extra_requirements }}"
+      environment:
+        # This module doesn't use the http_proxy as the env instead it uses the
+        # capitalized one
+        # reference https://docs.ansible.com/ansible/latest/modules/pip_module.html
+        HTTP_PROXY: "{{ http_proxy }}"
+      when: extra_requirments is defined and extra_requirements != []
+
+    - set_fact:
+        local_remote_config_file: "{{lookup('env','DEPLOYMENT_ROOT')}}/{{inventory_hostname}}/{{inventory_hostname}}.yml"
+        local_configs_dir: "{{lookup('env','DEPLOYMENT_ROOT')}}/{{inventory_hostname}}/configs"
+        python3: "{{lookup('env','DEPLOYMENT_ROOT')}}/env/bin/python"
```

Comment:
Don't use DEPLOYMENT_ROOT root from env, this should be set in the inventory

Reply:
DEPLOYMENT_ROOT is from the host perspective not the inventory perspective.  This should be the root of files to copy over to the system...note I am fine with having a different method

Reply:
Note also DEPLOYMENT_ROOT is set within the vctl deploy command as the directory where the host environment is specified.  If using outside that context this should be set at least until something else changes the way this works.

Reply:
I'll fix this, we're moving this to a variable which can be defined on the entire hosts group (not per host, since the logic knows to expect a subfolder per host).